packages/plugin-landing-page-graphql-playground/README.md

```diff
@@ -1,6 +1,6 @@
 # GraphQL Playground plugin
 
-> ⚠️ Note: this package will be completely unsupported when Apollo Server v4 is officially released. This package exists for migration purposes only. We do not intend to resolve security issues or other bugs with this package if they arise, so please migrate away from this to Apollo Server's default Explorer as soon as possible. ⚠️
+> ⚠️ Note: this package is unsupported. This package exists for v4 migration purposes only. We do not intend to resolve security issues or other bugs with this package if they arise, so please migrate away from this to Apollo Server's default Explorer as soon as possible. ⚠️
```

Comment:
I wonder if we should (a) link to https://www.apollographql.com/docs/apollo-server/previous-versions and (b) add something specific about this package there?

Reply:
Updated for (a) but this package seems a bit specific to be called out in (b). If you have a specific recommendation for (b) I'll consider it?